db/migrate/20230607161636_create_likes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
+      t.integer :post_id
```

Comment:
- [ ] The same here, In order to show that, the  `'author_id'` reference the User class. Kindly write the query like this so the foreign key name will be `author_id`:
```ruby
  create_table :likests do |t|
    t.references :author, foreign_key: { to_table: 'users' } 
    ...
```

- [ ] Take a look at the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). You should create the associations or foreign keys column using [references](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) column type also available as `belongs_to`. Your code should look similar to this:

```ruby
  create_table :likes do |t|
    t.timestamps
    t.references :post, foreign_key: true
  end

```
- This will create post_id and a corresponding index but students can also add column and index in separate commands.